index.css

```diff
@@ -0,0 +1,3 @@
+.header {
+  color: blue;
+}
```

Comment:
- [ ] [REQUIRED] Please, add styles to the `index.css` file as requested in project requirements. Currently it has no styles